htdocs/loan/card.php

```diff
@@ -192,7 +192,7 @@
 				$object->capital = $capital;
 				$object->nbterm = GETPOSTINT("nbterm");
 				$object->rate = price2num(GETPOST("rate", 'alpha'));
-				$object->insurance_amount = price2num(GETPOSTINT('insurance_amount'));
+				$object->insurance_amount = price2num(GETPOSTFLOAT('insurance_amount'));
```

Comment:
When using GETPOSTFLOAT, the price2num is not necessary. included into GETPOSTFLOAT.

Reply:
Correct @eldy ! Thanks